public/app/features/explore/state/datasource.ts

```diff
@@ -51,9 +51,8 @@ export function changeDatasource(
       })
     );
 
-    const queries = getState().explore[exploreId]!.queries;
```

Comment:
We use this only in  `if (options?.importQueries) {...` so I moved it below so we don't create queries variable if not needed.